### DISCUSSION TITLE
Java and Javascript failed to run

### DISCUSSION THREAD

**Discussion by chanpei9270** [thebaselab/codeapp#1183] [Q&A]
Hello everyone, I tried a simple hello world program but failed to run when it is executed the second time. The issue is the output from terminal tab is nothing. C and Python are OK without problem. Java and Javascript were actually OK right after installation but it will fail at the second time. I recorded the screen for your reference. Please see if you have any idea to identify the possible cause. Thanks a lot.

https://github.com/user-attachments/assets/23b83489-db9e-4bcc-a8c9-9f8282cb6915

**Comment by luni-moon** [comment]
To run your file in JS, run 
```
node default.js
```
Better practice for naming the main JS file is naming it index.js or main.js for future reference. 

For Java, look up documentation on how to run a Java programme.

The following StackOverflow answer may provide insight:
https://stackoverflow.com/a/42741643
